sky/shell/platform/android/org/domokit/sky/shell/PlatformViewAndroid.java

```diff
@@ -128,10 +133,36 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
         return super.onKeyDown(keyCode, event);
     }
 
-    public SkyEngine getEngine() {
+    SkyEngine getEngine() {
         return mSkyEngine;
     }
 
+    public void onPause() {
+        mSkyEngine.onAppLifecycleStateChanged(AppLifecycleState.PAUSED);
+    }
+
+    public void onResume() {
+        mSkyEngine.onAppLifecycleStateChanged(AppLifecycleState.RESUMED);
+    }
```

Comment:
I wonder if we should name these `pause` and `resume` because they're telling this view what to do.